styles/components/_tooltip.scss

```diff
@@ -0,0 +1,70 @@
+.tooltip {
```

Comment:
IMO, this can be ....
```
.tooltip {
  
  &.arrowleft {}

  &.arrowleft:after {}
}
```

or not?

Reply:
@srph I just left the company. You can assign this to someone.